packages/clerk-js/src/ui/components/OrganizationProfile/ActionConfirmationPage.tsx

```diff
@@ -145,7 +160,7 @@ const ActionConfirmationPage = withCardStateProvider((props: ActionConfirmationP
     try {
       await onConfirmation().then(() => wizard.nextStep());
     } catch (e) {
-      handleError(e, [], card.setError);
+      handleError(e, [], e => card.setError(getErrorMessage ? getErrorMessage(e) : e));
```

Comment:
Are we able to make use of the translateError function returned from useLocalizations here instead of adding the getErrorMessage prop utility? https://github.com/clerk/javascript/blob/main/packages/clerk-js/src/ui/localization/makeLocalizable.tsx#L74-L99

Reply:
I didn't know that we already had a utility for that! I've just found it while exploring the orgs components. 

Changed on https://github.com/clerk/javascript/pull/4123/commits/8f57b99faabad37eaae6e1cb90f9ef3049fdf59a